components/NavBar/navbar.module.css

```diff
@@ -1,99 +1,189 @@
-  height: 100%;
+  float: left;
 }
 
 .navBar li a:hover {
-  color: #e30464;
+  color: #49a82e;
 }
-.logo {
-  margin: 10px 0 5px 10px;
-  cursor: pointer;
+
+.navBarMenu {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #1d1283;
+}
+
+.navBarLogoLi {
+  padding: 12px 20px;
+}
+
+.navBar li.navBarLogoLi a {
+  margin: 0px;
+  padding: 0px;
+}
+
+.navBar li.navBarLoginLi {
+  float: right;
+}
+
+.navBar li.navBarLoginLi a {
+  padding: 12px;
 }
-.profilePic {
 
+.navBarLogin,
+.homeTab {
   display: none;
+}
+
+.btnLoginText {
+  border-radius: 6px;
+  padding: 5px;
+  background-color: #1d1283;
+  color: white;
+  cursor: pointer;
+  border: 2px solid white;
+}
 
-  margin-right: 1em;
-  width: 4em;
-  height: 4em;
+.userGreet {
+  color: #ffffff;
+  margin: 25px;
 }
-.profilePic>img {
-  width: 100%;
-  height: 100%;
+
+.userProfilePic {
+  width: 32px;
+  height: 32px;
   border-radius: 50%;
-  cursor: pointer;
+  display: inline;
+  vertical-align: middle;
 }
-.dropdownContent {
-  position: absolute;
-  right: 5px;
+
+.userGreetMsg {
+  display: inline;
+  vertical-align: middle;
+  margin-right: 10px;
 }
-.dropdownContentHide {
+
+.navBar li a.activeTab {
+  color: #49a82e;
+  text-decoration: underline;
+  text-decoration-thickness: 3px;
+  text-underline-offset: 10px;
+}
+
+.hamburger {
   display: none;
 }
-.dropdownContent {
-  position: absolute;
-  background-color: #f9f9f9;
-  min-width: 100px;
-  border: 1px solid rgba(0, 0, 0, 0.699);
-  z-index: 1;
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  background-color: #fff;
 }
-.dropdownContent a {
-  color: black;
-  padding: 5px 0;
+
+.githubLogo {
+  margin-left: 4px;
+}
+
+.header {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  padding-top: 5px;
+}
+
+.link {
+  padding: 10px;
+  margin-top: 2px;
   text-decoration: none;
-  text-align: center;
-  font-size: 18px;
-  border-bottom: 1px solid rgba(0, 0, 0, 0.26);
-  display: block;
+  font-weight: bold;
+  color: #041484;
+  cursor: pointer;
+  background: none;
+  border: none;
 }
-.dropdownContent a:hover {
-  background-color: #e4e2e2
+
+.active {
+  color: #e30464;
 }
 
+@media screen and (max-width: 970px) {
+  .navBar {
+    background: #1d1283;
+  }
 
-@media (max-width: 540px) {
-  .navBar li {
-    margin-right: 2rem;
+  .hamburger {
+    margin: 20px;
+    display: inline-block;
+    cursor: pointer;
   }
-  .loginBtn {
-    padding: .4em 1em;
+
+  .navBarMenu {
+    background: white;
+    display: none;
+    width: 100%;
+    padding: 10px 0px;
+    box-shadow: 0px 10px 15px rgba(0, 0, 0, 0.5);
   }
-}
 
-@media (max-width: 400px) {
-  .navBar {
-    font-size: .8rem;
-    height: 4rem;
-    margin-bottom: 15px;
+  .navBarMenu.active,
+  .homeTab {
+    display: block;
+  }
+
+  .navBar li a {
+    padding: 10px 40px;
+    text-align: left;
+    color: #1d1283;
+    text-decoration: none;
```

Comment:
No need to declare this property again, you have already declared on line 8 & 13.

Reply:
Yes, I missed it. Thanks for pointing it out.
Changes are implemented.

Reply:
@shubham-y